File: CPP/if_else.cpp
// Given an age, print "Adult" if age >= 18, or print "Teen"
#include<bits/stdc++.h>
using namespace std;

int main() {
    int age;
    cin >> age;

    if(age >= 18) {
        cout << "Adult" << endl;
    } else {
        cout << "Teen" << endl;
    }

    return 0;
}

File: Java/basics/IfElse.java
package basics;

import java.util.Scanner;

public class IfElse {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int age = scanner.nextInt();

        if(age >= 18) {
            System.out.println("Adult");
        } else {
            System.out.println("Teen");
        }

        scanner.close();
    }
}
